app/src/docs/components/DocsLayout/docsLayout.pcss

```diff
@@ -189,7 +189,7 @@
     picture img {
       display: block;
       margin: 0 auto;
-      max-width: 100%;
+      width: 100%;
```

Comment:
@fonty1 I did a quick look around and this didn't seem to affect any other images but was needed to make the vertical image reach the edges in mobile.

Reply:
Probably both are OK, I can't remember why I went for max-width but it seems fine to change.